Dockerfile-base-cpu

```diff
@@ -19,13 +19,17 @@ RUN apt-get update \
 		libtool \
 		git \
 		krb5-user \
-		g++-4.8 \
+		g++ \
+		cmake \
```

Comment:
Do we need this, given that we `pip install cmake==3.22.4` in Dockerfile-default-xxx?

Reply:
oh, I guess we do if ompi.sh needs it and is run as part of building the base images.